packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+* `system.client.concurrency_rate_limit.notification`: This event is fired when the framework is turned on in Preview mode. This event is fired when a specific client, IP address, Device token combination makes more than two concurrent requests. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, Device identifier, and the actual user if the user already has a valid session.
+
+### Check your rate limits with Okta Rate Limit headers
+
+The Rate Limit headers returned when client-based rate limiting is enabled are very similar to the headers returned today through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device combination rather than the org-wide Rate Limit values. Okta provides three headers in each response to report client-specific rate limits.
+
+> **Note:** If client-based rate limiting is in Preview or Disable mode, headers that are returned still reflect the org-wide rate limits.
+
+For client-specific rate limits, three headers show the limit that is being enforced, when it resets, and how close you are to hitting the limit:
+
+* `X-Rate-Limit-Limit` - The rate limit ceiling that is applicable for the current request to the specific client/IP/device identifier combination
+* `X-Rate-Limit-Remaining` - The number of requests left for the specific client/IP/device identifier combination during the current rate-limit window
+* `X-Rate-Limit-Reset` - The time when the rate limit resets, specified in [UTC epoch time](https://www.epochconverter.com/) for the specific client/IP/device identifier combination
+
+For example:
+
+```
+HTTP/1.1 200 OK
+X-Rate-Limit-Limit: 60
+X-Rate-Limit-Remaining: 35
+X-Rate-Limit-Reset: 1516307596
+```
+
+When a specific client/IP/device identifier combination exceeds either the per minute request limit (60 requests per minute) or the concurrent limit (two concurrent requests), the `/authorize` request returns an HTTP 429 error.
+
+### How to enable this feature
+
+<!-- need to see the UI to finish this section correctly and need more information on the NOTE right below-->
+
+> **Note:** For all orgs created after [Date 2020], this feature is automatically enabled.
+
+For existing orgs, to configure client-based rate limiting:
+
+> **Note:** The Admin Console is required for these steps. If you see **Developer Console** in the top left of the page, click it and select **Classic UI** to switch.
+
+1. From the Admin Console select **Reports** > **Rate Limits**, and then select the **Settings** tab.
```

Comment:
From the Admin console->settings->scroll down to client baseed rate limiting (section)